src/gpu/gpu_reorder_list.cpp

```diff
@@ -53,6 +54,7 @@ constexpr impl_list_item_t impl_list[] = REG_REORDER_P({
         GPU_REORDER_INSTANCE_INTEL(intel::ocl::ref_reorder_t::pd_t)    // slow but fits every use case
         GPU_REORDER_INSTANCE_NVIDIA(intel::ocl::cross_engine_reorder_t::pd_t)
         GPU_REORDER_INSTANCE_NVIDIA(nvidia::cudnn_reorder_t::pd_t)
+        GPU_REORDER_INSTANCE_NVIDIA(sycl::ref_reorder_t::pd_t)
```

Comment:
I bet this should be GENERIC version instead.

Reply:
Actually, after looking at what other options exist, I believe it should be GENERIC_SYCL.